frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/StaticEmbedSetupPane.tsx

```diff
@@ -0,0 +1,107 @@
+import { t } from "ttag";
+import { Tabs } from "metabase/ui";
+
+import type { ParametersSettingsProps } from "./ParametersSettings";
+import { ParametersSettings } from "./ParametersSettings";
+import type { AppearanceSettingsProps } from "./AppearanceSettings";
+import { AppearanceSettings } from "./AppearanceSettings";
+import type { OverviewSettingsProps } from "./OverviewSettings";
+import { OverviewSettings } from "./OverviewSettings";
+
+const TABS = {
+  Overview: "overview",
+  Parameters: "parameters",
+  Appearance: "appearance",
+};
+
+type StaticEmbedSetupPaneProps = OverviewSettingsProps &
+  ParametersSettingsProps &
+  AppearanceSettingsProps;
+
+export const StaticEmbedSetupPane = ({
+  activePane,
+  resource,
+  resourceType,
+  embedType,
+  token,
+  iframeUrl,
+  siteUrl,
+  secretKey,
+  params,
+  displayOptions,
+  previewParameters,
+  parameterValues,
+  resourceParameters,
+  embeddingParams,
+  onChangeDisplayOptions,
+  onChangeEmbeddingParameters,
+  onChangeParameterValue,
+  onChangePane,
+}: StaticEmbedSetupPaneProps): JSX.Element => {
+  return (
+    <div className="full flex">
```

Comment:
Let's use `Group` and `Stack` here - I know you probably didn't write this in the first place but it's a quick win